wiki/Community/Organisations/osu!_at_UCSD/en.md

```diff
@@ -0,0 +1,31 @@
+---
+tags:
+  - California
+  - club
+  - college
+  - San Diego
+  - school
+  - university
+---
+
+# osu! at UCSD
+
+**osu! at UCSD** is an osu! club run as a division of [Triton Gaming](https://discord.gg/tritongaming "Triton Gaming Discord server") at the University of California, San Diego (*UCSD*). Its goal is to give osu! players at or around UCSD a space to connect with each other and build a local community. They host regular meetups both in-person and online, as well as LAN and online osu! tournaments.
+
+See the [osu! at UCSD tournament index](/wiki/Tournaments/osu!_at_UCSD) for wiki articles about some of their tournaments.
```

Comment:
```suggestion
See the [osu! at UCSD tournament index page](/wiki/Tournaments/osu!_at_UCSD) for more information about previously held tournaments.
```

Reply:
previously held => their ?
the one in this PR isn't previous :^)

Reply:
mm ok

```suggestion
See the [osu! at UCSD tournament index page](/wiki/Tournaments/osu!_at_UCSD) for more information about their tournaments.
```